config/vufind/KohaRest.ini

```diff
@@ -165,4 +165,7 @@ extraFields = item-issue:acceptTerms:pickUpLocation
 [TransactionHistory]
 ; By default the loan history is disabled. Uncomment the following line to enable it.
 ;enabled = true
+; By default purging of all loan history is enabled. Uncomment the following line
+; to disable it:
+;purgeAll = false
```

Comment:
It seems a little dangerous to have this on by default -- libraries that rely on this data for statistics or other reasons might unexpectedly lose it. Would it be better for this to be an opt-in setting? Or at very least should we put a warning on the changelog?

(Practically speaking, there probably aren't too many libraries that have turned on the transaction history setting, which thus means that purging is not a risk... but it still seems like a situation that could potentially cause problems).

Reply:
For any normal use case the user should be able to clear their history since it's something they opted in for in the first place. This should not cause issues with statistics that would be still available, albeit maybe with less precision after anonymization. Also, this is the same setup as with Sierra, so if we end up turning purging off by default, we should do it for both.

Reply:
If it's an opt-in user feature, then I definitely don't object to keeping this as-is. Thanks for the clarification!